.github/workflows/prs.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Lint PR'
+on:
+  pull_request_target:
```

Comment:
```suggestion
  pull_request:
```

Reply:
https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/